projects/storefrontstyles/scss/_page-template.scss

```diff
@@ -0,0 +1,24 @@
+@import './cxbase/mixins';
+
+@import './layout/index';
+
+// The $page-template-blacklist can be used to prevent template CSS
+// being generated.
+$page-template-blacklist: () !default;
```

Comment:
Maybe we could rename these two variables. So like, $page-template-names. $page-templates-ignored?

Reply:
we agreed to keep using black vs whitelists.